src/main/java/uk/sky/cqlmigrate/CommandLine.java

```diff
@@ -0,0 +1,40 @@
+package uk.sky.cqlmigrate;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.google.common.base.Preconditions;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+public class CommandLine {
```

Comment:
I guess we felt that a main inside the CqlMigratorImpl wasn't the best place, and probably was better suited to be abstracted out into its own class for command line access. We'll move this into a new class as a separate commit